tools/bazar/fields/BazarField.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace YesWiki\Bazar\Field;
+
+use Psr\Container\ContainerInterface;
+use YesWiki\Core\Service\TemplateEngine;
+
+abstract class BazarField
+{
+    protected $services;
+
+    protected $type;         // 0
+    protected $id;           // 1
+    protected $label;        // 2
+    protected $size;         // 3
+    protected $minChars;     // 3
+    protected $maxChars;     // 4
+    protected $default;      // 5
+    protected $required;     // 8
+    protected $helper;       // 10
+    protected $readAccess;   // 11
+    protected $writeAccess;  // 12
+    protected $semanticPredicate; // 14
+
+    protected $recordId;     // How the field is identified in the Bazar record
+    protected $attributes;
+    protected $values;
+
+    // GLOBAL FIELDS
+    protected const FIELD_TYPE = 0;
+    protected const FIELD_ID = 1;
+    protected const FIELD_LABEL = 2;
+    protected const FIELD_SIZE = 3;
+    protected const FIELD_MIN_CHARS = 3;
+    protected const FIELD_MAX_CHARS = 4;
+    protected const FIELD_DEFAULT = 5;
```

Comment:
```suggestion
    protected const FIELD_DEFAULT = 5;
    protected const FIELD_LIST_LABEL = 6;
```
pour permettre de nous souvenir que le champ est déjà utilisé.